### PR TITLE
Gradient 🎨

### PR DESCRIPTION
**Pull request by moralesbang**
### Summary

Added `linear-gradient` to rent button, keeping animation compatibility.

### Screenshots

houston-rn-8

### Trello Card

https://trello.com/c/F8YU9AZY

### CODE REVIEW

File: src/app/screens/BookDetail/components/BookInfo/index.tsx [hunk 1]
```diff
@@ -74,16 +75,22 @@ function BookInfo() {
...
-            <Image style={styles.rentedIcond} source={rentedIcond} />
-          ) : (
-            <Text style={styles.rentButtonText}>RENT</Text>
-          )}
-        </AnimatedTouchableOpacity>
+        <LinearGradient
+          style={styles.rentButtonWrapper}
+          start={{ x: 0, y: 0 }}
+          end={{ x: 1, y: 0 }}
+          colors={['#00ADEC', '#39CCCD']}>
```
Comment (marianozicavo) [inline src/app/screens/BookDetail/components/BookInfo/index.tsx:82]: Move this array to colors.ts

File: src/app/screens/BookDetail/components/BookInfo/index.tsx [hunk 2]
```diff
@@ -74,16 +75,22 @@ function BookInfo() {
...
-          onPress={handleRent}>
-          {bookRented ? (
-            <Image style={styles.rentedIcond} source={rentedIcond} />
-          ) : (
-            <Text style={styles.rentButtonText}>RENT</Text>
-          )}
-        </AnimatedTouchableOpacity>
+        <LinearGradient
+          style={styles.rentButtonWrapper}
+          start={{ x: 0, y: 0 }}
```
Comment (marianozicavo) [inline src/app/screens/BookDetail/components/BookInfo/index.tsx:80]: Are the start and end props necessary? what happens if you remove them?

Comment (moralesbang) [reply src/app/screens/BookDetail/components/BookInfo/index.tsx:80]: Works well

Comment (marianozicavo) [reply src/app/screens/BookDetail/components/BookInfo/index.tsx:80]: Can you remove the start prop too?

Comment (moralesbang) [reply src/app/screens/BookDetail/components/BookInfo/index.tsx:80]: Yes

**Pull request merged**